js/index.js

```diff
@@ -19,6 +19,24 @@ for (i=0; i<skills.length; i++){
     skillsSection.appendChild(skill)
 }
 
+let githubRequest = new XMLHttpRequest ();
+const GITHUB_USERNAME = 'Asel-K'
+githubRequest.open("GET", `https://api.github.com/users/${GITHUB_USERNAME}/repos`);
+githubRequest.send(); 
+githubRequest.onload = function () {
+    let repositories = JSON.parse (this.response);
+    console.log ("response", repositories)
+
+let projectSection = document.getElementById ("projects");
+let projectList = projectSection.querySelector('ul')
+for(let i=0; i<repositories.length; i++){
+    let project = document.createElement('li');
+    project.innerHTML=`<a href=${`${repositories[i].clone_url}`}>${repositories[i].name}
+    </a>`
+    projectList.appendChild(project);
+}
+}
```

Comment:
Nitpicky indentation comment for lines 26-37, I think Line 30 can be indented by one to indicate that its in the `onload` block. I think in the assignment they ask for the code to be at the bottom of the file. But other than that, this all looks GREAT, and all requirements of the assignment have been met.